File: java/src/main/java/com/iteso/nintendo/attacks/SandAttack.java
package com.iteso.nintendo.attacks;

public class SandAttack implements Attack {
    static int damage = 30;
    static String name = "SandAttack";

    @Override
    public String attackName() {
        return this.name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

}

File: python/SandAttack.py
class SandAttack:
    def __init__(self):
        self.damage = None
        self.name = 'Sand Attack'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
